src/importers/postman/Parser.js

```diff
@@ -534,6 +551,24 @@ export default class PostmanParser {
         return request
     }
 
+    _putRequestsInGroup(group, ids, requests) {
+        let _group = group
+        for (let id of ids) {
+            let req = requests[id]
+            if (req) {
+                _group = _group.setIn(
+                    [
+                        'children', req.get('name') ||
+                        req.get('url')
+                    ],
+                    req
+                )
+            }
+        }
+
+        return _group
+    }
```

Comment:
```
if (req) {
   ...
}
```

is necessary for the edge case where the folder data from the collection is corrupted, and contains non existing request ids